conf.d/forgit.plugin.fish

```diff
@@ -2,10 +2,8 @@
 
 set INSTALL_DIR (dirname (dirname (status -f)))
 set -x FORGIT_INSTALL_DIR "$INSTALL_DIR/conf.d"
-set -x FORGIT "$FORGIT_INSTALL_DIR/bin/git-forgit"
```

Comment:
Unfortunately this line cannot change, this is auto-replaced in our homebrew script, so it should stay as it is! I would revert this entire section (5-8)

Reply:
@cjappl The line has just been moved (to line 8), why can't it be auto-replaced there now?

Reply:
Perhaps it could but it would need testing with the homebrew formula to make sure it still works. If this change can be done with keeping the original structure in place it's one less thing to check.

Reply:
I removed the 2nd commit modifying the variable.